viam-cartographer.go

```diff
@@ -555,12 +557,35 @@ func (cartoSvc *cartographerService) GetPointCloudMap(ctx context.Context) (func
 		return nil, ErrClosed
 	}
 
+	if cartoSvc.modularizationV2Enabled {
+		return cartoSvc.getPointCloudMapModularizationV2(ctx)
+	}
+
 	if !cartoSvc.localizationMode {
 		cartoSvc.mapTimestamp = time.Now().UTC()
 	}
 	return grpchelper.GetPointCloudMapCallback(ctx, cartoSvc.Name().ShortName(), cartoSvc.clientAlgo)
 }
 
+func (cartoSvc *cartographerService) getPointCloudMapModularizationV2(ctx context.Context) (func() ([]byte, error), error) {
+	chunk := make([]byte, chunkSizeBytes)
+	pc, err := cartoSvc.cartofacade.GetPointCloudMap(ctx, cartoSvc.cartoFacadeTimeout)
+	if err != nil {
+		return nil, err
```

Comment:
where is the test for this caes?

Reply:
[here](https://github.com/viamrobotics/viam-cartographer/blob/f5ae0a00f9487e2e41268baa2106de10e15ba7cd/viam-cartographer_internal_test.go#L489) I mock `cartoSvc.cartofacade.GetPointCloudMap` to return an error. Is this what you mean?